client-app/components/atoms/markdown-render/markdown-render.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div v-html="markdown"></div>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { marked } from "marked";
+import DOMPurify from "dompurify";
+
+const props = defineProps<{ src: string }>();
```

Comment:
Please, define props as argument, nor Generic. Type-only definition don't allow to use additional props options such as default value or required markers.